addons/crm/tests/test_crm_lead_convert.py

```diff
@@ -457,11 +457,10 @@ def test_mass_convert_deduplicate(self):
         duplicates if deduplicate is set to True. """
         lead_1_dups = self._create_duplicates(self.lead_1, create_opp=False)
         lead_1_final = self.lead_1  # after merge: same but with lower ID
-        lead_1_dups_partner = lead_1_dups[1]  # copy with a partner_id set but another email -> not correctly taken into account
 
         lead_w_partner_dups = self._create_duplicates(self.lead_w_partner, create_opp=False)
         lead_w_partner_final = lead_w_partner_dups[0]  # lead_w_partner has no stage -> lower in sort by confidence
-        lead_w_partner_dups_partner = lead_w_partner_dups[1]  # copy with a partner_id set but another email -> not correctly taken into account
+        lead_w_partner_dups_partner = lead_w_partner_dups[1]  # copy with a partner_id (with the same email)
```

Comment:
Meh, no. Don't change tests.

Reply:
(same reason as above, now, when we write on a lead, we write on the partner of the lead)

Reply:
Keep existing tests as they are as much as possible. Old data will exist in databases.

Reply:
but now it's impossible to have the same partner of another lead, but a different email :/
(so more lead will be detected as duplicated)

Reply:
> but now it's impossible to have the same partner of another lead, but a different email

I propose you to go on our production database.

Please think more about existing use cases and don't take best use case as granted.